cmd/podman/common/create.go

```diff
@@ -474,29 +474,29 @@ func DefineCreateFlags(cmd *cobra.Command, cf *ContainerCLIOpts) {
 	)
 	_ = cmd.RegisterFlagCompletionFunc(oomScoreAdjFlagName, completion.AutocompleteNone)
 
-	overrideArchFlagName := "override-arch"
+	archFlagName := "arch"
 	createFlags.StringVar(
-		&cf.OverrideArch,
-		overrideArchFlagName, "",
+		&cf.Arch,
+		archFlagName, "",
 		"use `ARCH` instead of the architecture of the machine for choosing images",
 	)
-	_ = cmd.RegisterFlagCompletionFunc(overrideArchFlagName, completion.AutocompleteNone)
+	_ = cmd.RegisterFlagCompletionFunc(archFlagName, completion.AutocompleteNone)
```

Comment:
If I remember correctly you added autocomplete functions for `arch` and `os`. It would be good if you add them here.

Reply:
Thanks for reminding me.